featuretools/computational_backends/feature_tree.py

```diff
@@ -25,16 +25,21 @@ def __init__(self, entityset, features, ignored=None):
             self.ignored = set([])
         else:
             self.ignored = ignored
+
+        self.feature_hashes = set([f.hash() for f in features])
+
         all_features = {f.hash(): f for f in features}
         feature_deps = {}
         for f in features:
             deps = f.get_deep_dependencies(ignored=ignored)
             feature_deps[f.hash()] = deps
             for dep in deps:
                 all_features[dep.hash()] = dep
-                feature_deps[dep.hash()] = dep.get_deep_dependencies(ignored=ignored)
+                feature_deps[dep.hash()] = dep.get_deep_dependencies(
+                    ignored=ignored)
         self.all_features = list(all_features.values())
         self.feature_deps = feature_deps
+        self.feature_dependents = self._build_dependents()
```

Comment:
why break this out into a different function? can it be combined into the logic above that calculates `self.feature_deps`?

Reply:
we would have to change the above logic to accommodate it, since we rely on get_deep_dependencies() but we don't have an equivalent get_deep_dependents() method. It would just have to be a bit more complicated as a queue/search/while loop instead of just the two for loops. But yeah might make it look more uniform

Reply:
Nevermind i think i can make it work